src/api/clients/IOClients/apps/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './ABTester'
+export * from './Billing'
+export * from './Builder'
+export * from './Rewriter'
+export * from './Sponsor'
+export * from './Tester'
+export * from './ToolbeltConfig'
+export * from './ToolbeltTelemetry'
```

Comment:
Plugins will not need those two clients, they are toolbelt internals

Reply:
Just remove this export will be enough ? Bexause they will be located at `api`folder. Do you think that creating the same folder inside `src/lib` it's a better approach ?